creator-node/src/config.js

```diff
@@ -502,6 +502,12 @@ const config = convict({
     env: 'snapbackJobInterval',
     default: 1800000 // 30min
   },
+  snapbackUsersPerJob: {
+    doc: 'Maximum number of users to process in each SnapbackSM job',
+    format: 'nat',
+    env: 'snapbackUsersPerJob',
+    default: 1000
```

Comment:
im gonna ask 🌚  how did we come up with this #?

Reply:
it's somewhat magical haha but I notice it takes <10 seconds to run 1000 users, and queues should be no more than 30 seconds according to the Bull docs, so 1000 users gives wiggle room for errors and retries. the number is up for debate tho that's why I made it configurable